amethyst_audio/src/sink.rs

```diff
@@ -59,3 +59,62 @@ impl AudioSink {
         self.sink.stop();
     }
 }
+
+// q: for AudioSink, do we want to test each of these apis as a sanity check?
```

Comment:
I don't think it's necessary; it may be a better idea to do one of the following in a separate PR:

* implement `Deref` and `DerefMut` with `Sink` as the target type, or
* use the [`delegate`](https://crates.io/crates/delegate) crate -- not really liking this option, but worth listing

Reply:
Oops that was not meant for review. I was gonna ask in Discord or something. Will remove the comment.